docs/markdown/Running-Meson.md

```diff
@@ -6,7 +6,7 @@ short-description: Building a project with Meson
 
 There are two different ways of invoking Meson. First, you can run it
 directly from the source tree with the command
-`/path/to/source/meson.py`. Meson may also be installed in which case
+`/path/to/source/meson.py`. Second, Meson may also be installed in which case
```

Comment:
When changing from "Meson may also be installed"

shouldn't it become

"Second, Meson may be installed"?

Reply:
I would say no, but I am not sure 100%.

Reply:
I don't think "second" is really the right word here. Maybe "Alternatively, Meson may be installed, in which case…"